src/types/EpicTypes.ts

```diff
@@ -160,4 +162,6 @@ export interface EpicTest extends Test<EpicVariant> {
     campaignId?: string;
 
     controlProportionSettings?: ControlProportionSettings;
+
+    isSuperMode?: boolean;
```

Comment:
is this field just used for setting the label?

Reply:
yeah that's all it's used for atm